diff --git a/packaging/CMakeLists.txt b/packaging/CMakeLists.txt
index 2c5b038..baeaba4 100644
--- a/packaging/CMakeLists.txt
+++ b/packaging/CMakeLists.txt
@@ -32,7 +32,7 @@ install(TARGETS Halide Halide_Generator Halide_LanguageOptions
         INCLUDES DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
 
 if (TARGET Halide_Adams2019)
-    install(TARGETS Halide_Adams2019 Halide_Li2018 Halide_Mullapudi2016
+    install(TARGETS Halide_Anderson2021 Halide_Adams2019 Halide_Li2018 Halide_Mullapudi2016
             EXPORT Halide_Interfaces
             LIBRARY DESTINATION ${Halide_INSTALL_PLUGINDIR} COMPONENT Halide_Runtime
             NAMELINK_COMPONENT Halide_Development)
diff --git a/src/autoschedulers/anderson2021/DefaultCostModel.cpp b/src/autoschedulers/anderson2021/DefaultCostModel.cpp
index 3eede59..75c0723 100644
--- a/src/autoschedulers/anderson2021/DefaultCostModel.cpp
+++ b/src/autoschedulers/anderson2021/DefaultCostModel.cpp
@@ -321,7 +321,7 @@ void DefaultCostModel::evaluate_costs() {
     }
 
     cursor = 0;
-    cost_per_stage_ptrs.clear();
+    // cost_per_stage_ptrs.clear();
 }
 
 void DefaultCostModel::load_weights() {
